crates/configuration/src/relaychain.rs

```diff
@@ -44,42 +44,52 @@ pub struct RelaychainConfig {
 }
 
 impl RelaychainConfig {
+    /// The chain name.
     pub fn chain(&self) -> &Chain {
         &self.chain
     }
 
+    /// The default command used for nodes.
     pub fn default_command(&self) -> Option<&Command> {
         self.default_command.as_ref()
     }
 
+    /// The default container image used for nodes.
     pub fn default_image(&self) -> Option<&Image> {
         self.default_image.as_ref()
     }
 
+    /// The default resources limits used for nodes.
     pub fn default_resources(&self) -> Option<&Resources> {
         self.default_resources.as_ref()
     }
 
+    /// The default database snapshot location that will be used for state.
     pub fn default_db_snapshot(&self) -> Option<&AssetLocation> {
         self.default_db_snapshot.as_ref()
     }
 
+    /// The default args that will be used to launch the node command.
     pub fn default_args(&self) -> Vec<&Arg> {
         self.default_args.iter().collect::<Vec<&Arg>>()
     }
 
+    /// The location of an pre-existing chain spec for the relaychain.
     pub fn chain_spec_path(&self) -> Option<&AssetLocation> {
         self.chain_spec_path.as_ref()
     }
 
-    pub fn random_minators_count(&self) -> Option<u32> {
+    // TODO: description?
+    pub fn random_nominators_count(&self) -> Option<u32> {
         self.random_nominators_count
     }
 
+    // TODO: description?
```

Comment:
```suggestion
    /// The maximum number of nominations to create per nominator, as default we use `24` as is set in `kusama`.
```

Reply:
At the moment. the max_nominations is not required, hence the default is None, should we change it ?

Reply:
No, we can leave as is now and we can manage those cases in the orchestrator. Since is a very specific use-case.
Thx!

Reply:
I updated to : ```The maximum number of nominations to create per nominator.``` (removed the text about the default given we put it to None)